tests/Integration/RequestIntegrationTest.php

```diff
@@ -8,52 +8,77 @@
 use Psr\Http\Message\ServerRequestInterface;
 use Sentry\Event;
 use Sentry\Integration\RequestIntegration;
+use Sentry\Options;
 use Zend\Diactoros\ServerRequest;
 use Zend\Diactoros\Uri;
 
 final class RequestIntegrationTest extends TestCase
 {
     /**
-     * @dataProvider invokeDataProvider
+     * @dataProvider invokeUserContextPiiDataProvider
      */
-    public function testInvoke(array $requestData, array $expectedValue): void
+    public function testInvokeWithRequestHavingIpAddress(bool $pii, array $expectedValue): void
```

Comment:
Can we rename the variable `$pii` to something that better describes its purpose (e.g. `$shouldSendPii`)?